src/config/dynamic-theme-fixes.config

```diff
@@ -2439,6 +2439,24 @@ IGNORE IMAGE ANALYSIS
 
 ================================
 
+gls-pakete.de
+
+CSS
+.tracking--status .status-box::before, .tracking--status .status-box::after {
+    z-index: 0 !important;
+}
+
+.tracking--status .status-box.status--complete.status--lastcomplete .status-box--tooltip, .tracking--status .status-box.status--current .status-box--tooltip {
+    -webkit-transform: none !important;
```

Comment:
You can drop the `-webkit-` as all supported browsers and minimum version already drop this.